home/templates/home/blocks/tile_column_block.html

```diff
@@ -1,110 +1,220 @@
+                {% if self.text_col2 %}
+                  <span class="card-text text-muted">{{ self.text_col2 }}</span>
+                {% endif %}
+                </div>
+              </a>
+            </div>
+            <div class="card tile-block__tile">
+            {% if self.internal_link_col3 %}
+              <a href="{{ self.internal_link_col3.url }}">
+            {% elif self.external_link_col3 %}
+              <a href="{{ self.external_link_col3 }}">
             {% endif %}
+
+              {% if self.image_col3 %}
+                {% image self.image_col3 fill-300x268 class="card-img-top" %}
+              {% else %}
+                <img src="{% static 'images/pink-bg.png'%}" class="card-img-top" />
+              {% endif %}
+
+                <div class="card-body">
+                {% if self.title_col3 %}
+                  <h3 class="block-title">{{ self.title_col3 }}</h3>
+                {% endif %}
+                {% if self.text_col3 %}
+                  <span class="card-text text-muted">{{ self.text_col3 }}</span>
+                {% endif %}
+                </div>
+              </a>
             </div>
-          </a>
-        </div>
-      </div>
+            <div class="card tile-block__tile">
+              {% if self.internal_link_col4 %}
+                <a href="{{ self.internal_link_col4.url }}">
+              {% elif self.external_link_col4 %}
+                <a href="{{ self.external_link_col4 }}">
+              {% endif %}
 
-      <div class="col-lg-3 d-flex align-items-stretch">
-        <div class="card tile-block__tile">
-        {% if self.internal_link_col2 %}
-          <a href="{{ self.internal_link_col2.url }}">
-        {% elif self.external_link_col2 %}
-          <a href="{{ self.external_link_col1 }}">
-        {% endif %}
+              {% if self.image_col4 %}
+                {% image self.image_col4 fill-300x268 class="card-img-top" %}
+              {% else %}
+                <img src="{% static 'images/blue-bg.png'%}" class="card-img-top" />
+              {% endif %}
+
+                <div class="card-body">
+                {% if self.title_col4 %}
+                  <h3 class="block-title">{{ self.title_col4 }}</h3>
+                {% endif %}
+                {% if self.text_col4 %}
+                  <span class="card-text text-muted">{{ self.text_col4 }}</span>
+                {% endif %}
+                </div>
+              </a>
+            </div>
+          </div>
 
-          {% if self.image_col2 %}
-            {% image self.image_col2 fill-300x268 class="card-img-top" %}
-          {% else %}
-            <img src="{% static 'images/light-blue-bg.png'%}" class="card-img-top" />
+<!-- Desktop version -->
+      <div class="row d-none d-lg-flex">
+        <div class="col-lg-3 d-flex align-items-stretch">
+          <div class="card tile-block__tile">
+          {% if self.internal_link_col1 %}
+            <a href="{{ self.internal_link_col1.url }}">
```

Comment:
@fuzzylogic2000 maybe here is better, all the cols are number to ensure notmore than 4 are added but becasue of that I couldn't figure out how to loop through, instead i'm duplicating code whish I don't like